js/src/mpl.js

```diff
@@ -253,7 +189,21 @@ figure.prototype.handle_rubberband = function(msg) {
 
 figure.prototype.handle_figure_label = function(msg) {
     // Updates the figure title.
-    this.header.textContent = msg['label'];
+    this.figure_label = msg['label'];
+    this.update_header();
```

Comment:
I think the message info should be in a separate status bar rather than adding text to the title.

Reply:
It looks better like that indeed